src/components/Banner.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+import Slider from './Slider';
+
+const Banner: React.FC = () => {
+    const [images, setImages] = useState<string[]>([]);
+
+    useEffect(() => {
+        fetch('http://localhost:5000/images')
```

Comment:
```suggestion
        fetch(`${process.env.API_HOST}/images`)
```
Em đặt http://localhost:5000 ra biến môi trường nha